client/src/components/CategoryProducts/CategoryProducts.tsx

```diff
@@ -57,18 +52,18 @@ const CategoryProducts = ({ ...props }: IProps) => {
   if (!data) return <div>nodata</div>;
 
   const renderCard = () => {
-    const MapData = new Map();
-    data.pages.flat().forEach((item) => MapData.set(item.id, item));
-    return Array.from(MapData, ([, value]) => value).map((item) => (
-      <Card
-        linkId={item.id}
-        key={item.id}
-        bgColor="primary"
-        src={item.productImage[0].url}
-        price={item.price}
-        title={item.title}
-      />
-    ));
+    return data.pages
+      .flat()
+      .map((item) => (
```

Comment:
이거 flatMap 으로 바꿔도 되지 않나요?? 아마??

```suggestion
      .flatMap((item) => (
```

Reply:
오와 이거 뭐야 첨알았어...

Reply:
잉... 안되네용,,,ㅠ

item이 배열 반환이어서 flatMap(item => item.map())
요런식으로 해야 오류가 안뜨네용...

평탄화 작업을 먼저 해주느냐 안해주느냐의 차이인듯..